lib/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -1,34 +1,11 @@
-#ifndef TRITON_CONVERSION_TRITONGPU_TO_LLVM_UTILITY_H
-#define TRITON_CONVERSION_TRITONGPU_TO_LLVM_UTILITY_H
+#ifndef TRITON_CONVERSION_TRITON_GPU_TO_LLVM_UTILITY_H_
```

Comment:
Do we have to change the style of all the header guards?

The original one follows the mlir/llvm style, check here https://github.com/llvm/llvm-project/blob/main/mlir/include/mlir/AsmParser/AsmParserState.h#L9

```c++
#ifndef MLIR_ASMPARSER_ASMPARSERSTATE_H
#define MLIR_ASMPARSER_ASMPARSERSTATE_H
```
It seems the original style is ok.

Reply:
Yeah, but originally some files have _ some without. So I just unify them into one of them... I will unify them.

Reply:
Yeah, that makes sense.  In the current codebase, we tend to follow the LLVM code style I think, if there is multiple choice, the more LLVM-style-like one might be better.